packages/integration_test/README.md

```diff
@@ -83,7 +83,7 @@ flutter drive \
 ### Web
 
 Make sure you have [enabled web support](https://flutter.dev/docs/get-started/web#set-up)
-then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#6b-web)
+then [download and run](https://docs.flutter.dev/cookbook/testing/integration/introduction#5b-web)
```

Comment:
```suggestion
then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#5b-web)
```

Reply:
Actually I think `docs.flutter.dev` is correct, you were right the first time.

Reply:
Oh, thanks ❤️ I fixed it 👍